app/controllers/projects_controller.rb

```diff
@@ -343,6 +343,6 @@ def return_context
     render_404 && return if @project.blank?
     return unless @project.deleted?
     flash[:alert] = 'The project you requested had been deleted.'
-    redirect_to user_path(@user)
+    check_redirect "/#{@user.username}"
```

Comment:
yeah I know... I'm talking about the master branch here. I went through the warnings but couldn't find that line so I was wondering if it's there and I couldn't find it